### ISSUE TITLE
Improvements asked by Forence for deliverable "Early detection system for IAS - Version 1"

### ISSUE THREAD

**Issue by damianooldoni** [riparias/.github#1]
These styling improvements are proposed by Florence:

- [x]  Entitle the document in the same way as in the grant agreement, so that our controller understands which official deliverable it is: « Early detection system for IAS- Version 1 » (A1)
- [x] Add a short intro section recalling what this official deliverable is about and its role in the project
- [x] Add a small legend under the graph explaining the abbreviations / technical « jargon » used (i.e Fulcrum ; DwC ; GBIF…)  
- [x] Use « LIFE RIPARIAS » everywhere as it is the official acronym of the project (rather than « RIPARIAS »)
- [x] Instead of using « we … », use an impersonal style (e.g. instead of We automatically harvest observations …use for instance « In the framework of the project, observations are automatically harvested…)
- [x] Recall that this early detection system is meant to be a first version and explain briefly what is meant to be improved in the upcoming version n°2 (due by 30/09/2023)
- [x] Put your text in the LIFE RIPARIAS template  as this will be annexed to your Annual report
[LIFE RIPARIAS Template 1-Word- with cover page.docx](https://github.com/riparias/.github/files/7692015/LIFE.RIPARIAS.Template.1-Word-.with.cover.page.docx)

**Comment by damianooldoni** [comment]
@niconoe: I assigned myself for tackling these comments. Still, your content related improvements (see https://github.com/riparias/early-warning-webapp/issues/51 ) are welcome!

The last checkbox is something I will do at the very end (with a link to the version of the README.md the text is taken from)

**Comment by niconoe** [comment]
Thanks a lot Damiano!

I guess at this stage we can do all those improvements on the current page? If at some point we realize that we need two different documents for difference audience (because the project controller wants to see drastically different things than a contributing developer), we'll still be able to split things into two different documents.

**Comment by damianooldoni** [comment]
Indeed. As I wrote, we still have to apply the RIPARIAS template (which is a MS Word template!) so I can copy paste only part of this document and put the right title as Florence asked. So, I would not change the title of this document: 
**this document is more than a deliverable**

**Comment by damianooldoni** [comment]
> Recall that this early detection system is meant to be a first version and explain briefly what is meant to be improved in the upcoming version n°2 (due by 30/09/2023)
Done in https://github.com/riparias/early-warning-webapp#next-steps

**Comment by damianooldoni** [comment]
About:

> Add a short intro section recalling what this official deliverable is about and its role in the project


## Introduction

This deliverable aims to describe the work done to build a first version of an early detection system for IAS upon an LIFE RIPARIAS aggregated database. The goal of the early detection system is to:

1. display the information stored in LIFE RIPARIAS aggregated database
2. allow managers to be rapidly aware of new occurrences of IAS

**Comment by damianooldoni** [comment]
First and last points of Florence's comments are to be solved within MS Word.

**Comment by damianooldoni** [comment]
MS Word document is ready. Contact Florence to check everything is ok. If yes, close issues. Other comments from NEEMO should be reported in a new issue.
